data/xml/W19.xml

```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
       <abstract>The tutorial will introduce a set of very useful statistical tests for conducting analyses in the research areas of Machine Translation (MT) and Translation Studies (TS). For each statistical test, the presenter will: 1) introduce it in the context of a common research example that pertains to the area of MT and/or TS 2) explain the technique behind the test and its assumptions 3) cover common pitfalls when the test is applied in research studies, and 4) conduct a hands-on activity so that attendees can put the knowledge acquired in practice straight-away. All examples and exercises will be in R. The following statistical tests will be covered: t-tests (both parametric and non-parametric), bootstrap resampling, Pearson and Spearman correlation coefficients, linear mixed-effects models.</abstract>
     </paper>
   </volume>
+  <volume id="77" ingest-date="2019-10-01">
+    <meta>
+      <booktitle>Proceedings of the Fifth International Conference on Dependency Linguistics (Depling, SyntaxFest 2019)</booktitle>
+      <url>W19-77</url>
+      <editor><first>Kim</first><last>Gerdes</last></editor>
+      <editor><first>Sylvain</first><last>Kahane</last></editor>
+      <publisher>Association for Computational Linguistics</publisher>
+      <address>Paris, France</address>
+      <month>27–28 August</month>
+      <year>2019</year>
+    </meta>
+    <frontmatter></frontmatter>
+    <paper id="1">
+      <title><fixed-case>S</fixed-case>yntax<fixed-case>F</fixed-case>est 2019 Invited talk - Inductive biases and language emergence in communicative agents</title>
+      <author><first>Emmanuel</first><last>Dupoux</last></author>
+      <pages>1–1</pages>
+      <url>W19-7701</url>
+    </paper>
+    <paper id="2">
+      <title><fixed-case>S</fixed-case>yntax<fixed-case>F</fixed-case>est 2019 Invited talk - Transferring <fixed-case>NLP</fixed-case> models across languages and domains</title>
```

Comment:
Should invited talks get anthology entries?

Reply:
I guess, if there's an associated PDF.